src/Microsoft.TestPlatform.CoreUtilities/Utilities/StringExtensions.cs

```diff
@@ -17,5 +19,28 @@ public static string AddDoubleQuote(this string value)
         {
             return "\"" + value + "\"";
         }
+
+        public static void AppendToStringBuilderBasedOnMaxLength(this string data, StringBuilder result)
```

Comment:
Can we write comment as well for this method?

Reply:
Done.